CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,104 @@
+# Code of Conduct
+
+To help us deliver great features and support the Renovate Open Source project we ask that you:
+
+- are polite
+- pay attention to details
+- keep in mind that most maintainers are volunteers
+- are respectful of the time and effort of the maintainers
+
+## Our priorities
+
+We want to keep this project sustainable.
+This means we support our maintainers and contributors, who spend their free time to help others.
+
+Maintainers getting stressed is a big threat to Open Source projects, like ours.
+Stressed maintainers quit, or reduce their time spent on the project.
+Often a few users behave badly, where most users are nice.
+
+We want to avoid maintainers getting stressed out by bad behavior from contributors.
+That's why we have these rules.
+
+## Politeness
+
+Sadly, it's common in Open Source projects for a few users to behave in an aggressive and rude way.
+A user might say something like: "You should have fixed this bug already!", or "Why am I still waiting for this feature?".
+We do not allow this kind of behavior.
+
+We expect basic politeness, do not act rude.
+For example: it is okay if you ask a question and do not thank us afterwards.
+But avoid writing mean comments like: "Pity the documentation didn’t say that." or "Thanks for nothing.".
+
+## Respect the time of those who help you
+
+Respect goes both ways, but time is limited.
+When you ask for help, please remember that the maintainer's time is valuable.
+We get many questions each week and do our best to answer each one.
+To get the help you need, please be prepared to give detailed logs or descriptions of your issues.
+If you do not want to spend the effort giving us enough information, it's likely you will not get the help you need.
+
+Remember, most of the support provided by our team, including the Mend.io staff, is _unpaid_.
+
+## Blocking and unblocking
```

Comment:
I found a section about blocking in the [`contributing.md` file on your branch](https://github.com/renovatebot/renovate/blob/docs/coc/.github/contributing.md):

> ## Rate Limiting of Support Requests through Temporary Blocking
> 
> To ensure that the Renovate maintainers don't burn out from dealing with unfriendly behavior, those who display a bad attitude when asking for or receiving support in the repo will be rate limited from further requests through the use of temporary blocking.
> The duration of the temporary block depends on how rude or inconsiderate the behavior is perceived to be, and can be from 1-30 days.
> 
> If you have been blocked temporarily and believe that it is due to a misunderstanding, or you regret your comments and wish to make amends, please reach out to the lead maintainer Rhys Arkins by email with any request for early unblocking.
> If/once you are unblocked, you should edit or delete whatever comment lead to the blocking, even if you did not intend it to be rude or inconsiderate.
> Long emails or apologies are undesirable - the maintainers are busy and want to be able to help as many users as possible with the time they have available.

I think the parts that are still relevant should be edited and moved to the Code of Conduct. Then add a link to the Code of Conduct in place of the old text.

It's probably good to mention how to get "unbanned/unblocked" in the Code of Conduct. And how long/short you want the apology to be.